### DISCUSSION TITLE
widget.NewTableWithHeaders() has bug？

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#5186] [Q&A]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

For example, a table has 10 entries, of which 1 is in state A, 2 are in state B, 3 are in state C, and 3 are in state D
So if the user clicks on state C and then clicks on state D, because both state C and state D have 3 pieces of data, the table content will not refresh.
This bug is:
If the same amount of data is given to the table, it will not refresh and will only refresh by clicking on the next cell

### Describe the solution you'd like to see.

This problem only occurs when the amount of data is the same.

**Comment by andydotxyz** [comment]
Please do not use a feature request form to submit a bug.
If you are asking a question use the discussion feature here or one of our social media channels.

It is unclear what bug you are reporting because tapping on a table will not cause it to refresh, it sounds like you have some code internally performing some data manipulation that is missing from this bug/feature report.

**Comment by Jacalz** [comment]
I'm moving this to a discussion. Let's use the bug report template with sufficient information when, and if, a bug has been found.
